### PR TITLE
refactor(RHTAPREL-773): replace advisory key with releaseNotes

### PR DESCRIPTION
**Pull request by johnbieren**
Update the create-advisory-internal-request task to replace the references to the advisory key in the data.json with the releaseNotes key.

### CODE REVIEW

**Comment by johnbieren** [comment]
/test release-pipelines-e2e-suite
rbac issues

File: tasks/check-data-keys/README.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 Tekton task to check that all required information is present in the data file in order to make a
 call to the specified system(s). If any required keys are missing, the task will fail.
 
-For example, if `advisory` is passed as a system and the data file does not have all the required
-advisory keys, the task will fail.
+For example, if `releaseNotes` is passed as a system and the data file does not have all the required
```
Comment (mmalina) [inline tasks/check-data-keys/README.md:6]: I understand that you're just replacing "advisory" with "releaseNotes". But I am confused about the word "system" here. The first paragraph says "in order to make a
call to the specified system(s)". And the last paragraph says that advisory/releaseNotes is the system. How do you make a call to releaseNotes? What does it mean? So maybe just the first paragraph needs some rewording? Any idea what it should say?

File: tasks/check-data-keys/check-data-keys.yaml [hunk 2]
```diff
@@ -32,17 +32,16 @@ spec:
         set -ex
 
         KEYS_JSON='{
-            "advisory": [
-                "repo",
```
Comment (mmalina) [inline tasks/check-data-keys/check-data-keys.yaml:36]: Is the repo now set globally somewhere? Hardcoded in the internal cluster task? I assume this was the gitlab repo containing the advisories, right?

Comment (johnbieren) [reply tasks/check-data-keys/check-data-keys.yaml:36]: https://issues.redhat.com/browse/RHTAPREL-815

File: tasks/check-data-keys/README.md (hunk 1 above)
Comment (johnbieren) [reply tasks/check-data-keys/README.md:6]: If you have a better word for [releaseNotes, fbc, images, mapping, maybe I am missing something else?], I am happy to change it. System was just the word I came up with for it

File: tasks/check-data-keys/check-data-keys.yaml (hunk 2 above)
Comment (mmalina) [reply tasks/check-data-keys/check-data-keys.yaml:36]: Awesome, thanks!

File: tasks/check-data-keys/README.md (hunk 1 above)
Comment (mmalina) [reply tasks/check-data-keys/README.md:6]: I'm fine with system. But with this meaning, I am not sure about "make a call to the specified system(s)" (I guess I should have commented there and not here, but at first I thought this second occurrence of "system" is maybe the one to change).

So maybe replace "make a call to" with "use" or "process"? Like: "Tekton task to check that all required information is present in the data file in order to use the specified system(s)."

Comment (davidmogar) [reply tasks/check-data-keys/README.md:6]: I would have called it `feature` maybe and remove the `make a call` comment.

Something like:
```
Tekton task to check that all required information is present in the data file in order to make use of the specified features. If any required keys are missing, the task will fail.
```

Comment (johnbieren) [reply tasks/check-data-keys/README.md:6]: I'm not sold on using the word feature, I think it is a very commonly used word and is actually overused. I do like Martin's suggestion though (that it seems David agrees with) and pushed a change with it

Comment (mmalina) [reply tasks/check-data-keys/README.md:6]: Sounds good to me, thanks.

**Review by mmalina — APPROVED** [review]

**Pull request merged**